include/logging/log_rpc.h

```diff
@@ -117,6 +117,32 @@ int log_rpc_fetch_history(log_rpc_history_handler_t handler);
  */
 int log_rpc_get_crash_log(size_t offset, char *buffer, size_t buffer_length);
 
+/**
+ * @brief Gets current buffer usage threshold.
+ *
+ * The function fetches the current buffer usage signal threshold (percent).
+ *
+ * @returns The current threshold.
+ */
+uint8_t log_rpc_get_buffer_usage_signal_threshold(void);
```

Comment:
```suggestion
uint8_t log_rpc_get_history_usage_signal_threshold(void);
```
Same for the other API + enum log_rpc_group.h.
This is for consistency as other functions for this functionality use `_history_` instead of `_buffer_`. I switched to "history" because "buffer" is too ambiguous (there are multiple buffers involved in logging).